doc/conf.py

```diff
@@ -179,12 +184,29 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
+    "dark_css_variables": {
+        "color-announcement-background": "#935610",
```

Comment:
This is returning a red for me (and the `light_css_variables` equivalent a green), but in the example screenshot you include it's a light orange. Could you confirm where these colors are shown ?

Reply:
Humm that's weird haha. I picked the color from [blueprintjs](https://blueprintjs.com/docs/#core/colors) and just double-checked, everything should be alright. 
It seems ok in too ; does yours say otherwise?
![Screenshot from 2022-09-27 22-42-06](https://user-images.githubusercontent.com/13835654/192631237-68e4a2d1-b829-4fdb-b4a0-f0666bee151e.png)

Reply:
Ah, you're totally right ! Somehow my hex codes were getting converted inappropriately. Sorry !!!

Maybe one idea : Could we replace #935610 with (a shade near) #3499cd to match the nilearn logo ? So "blue" is stable and "orange" is unstable ?

Reply:
Ah yes I didn't mean exactly that code, but I think something near it would work well -- and it's perhaps a bit clearer than brown that it's stable, since there's more contrast than between brown and orange.

Small details, though, so feel free to overrule.

Reply:
Oh maybe I didn't understand your comment then: I don't think we need to tweak the color for stable version because this banner will only be displayed if you are not looking at the stable version.

Reply:
Sorry for being unclear ! I was preempting a bit : My understanding was that we were planning to add back in a stable banner, but this was likely a misunderstanding on my part ! I don't (personally) think it's necessary. 
If in fact we were going to add it back, I thought this CSS was intended for that. Sorry for the noise, though, and thanks again for this change !!